src/components/autocomplete/OfficerNameAutocomplete/index.tsx

```diff
@@ -46,3 +46,7 @@ export const OfficerNameAutocomplete = (props: OfficerAutocompleteProp) => {
     />
   );
 };
+
+OfficerNameAutocomplete.defaultProps = {
```

Comment:
can drop defaultProps with line 20

Reply:
Weird enough, eslint gave me an error for not having defaultProps even tho I already set a default value lmao...

Reply:
You know any way of fixing that? Idk why it's happening lol. Worst case scenario I might have to turn that off by line